fingerprinting-protections/init.js

```diff
@@ -0,0 +1,29 @@
+function init(window) {
```

Comment:
I think document that these values are being set on purpose to be 'outlier' values on purpose so the test can verify in a platform agnostic way.